tests/Schema/Keywords/MultipleOfTest.php

```diff
@@ -21,6 +21,7 @@ public function validDatasets(): array
             [10, .5],
             [9.9, .3],
             [.94, .01],
+            [1, .3333333333],
```

Comment:
Despite the fact it is formally correct according to current epsilon I'd rather avoid fix this as a valid behaviour

Reply:
yep, just drop it